### PR TITLE
Tracklist UI

### PR DESCRIPTION
**Pull request by ar7urs**

### CODE REVIEW

File: src/components/Section/Section.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import classnames from 'classnames'
+import styles from './Section.module.scss'
+
+const Section = ({ className, children }) => (
+  <section className={classnames(styles.section, className)}>
```
Comment (apata) [inline src/components/Section/Section.js]: Add the section className as well for better bulma semantics (<section className={classnames("section", styles.section, className}>

File: src/components/TracklistsCard/TracklistsCard.js [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
-const TracklistsCard = ({ text, image, title, twitter, slack }) => (
-  <div className={styles.card}>
-    <img src={image} />
+const TracklistsCard = ({ url, text, image, title }) => (
+  <Link className={styles.card} to={url}>
+    <img alt={title} src={image} />
     {title && <p className={styles.title}>{title}</p>}
-    {text && <p>{text}</p>}
-  </div>
+    {text && <p className={styles.content}>{text}</p>}
```
Comment (apata) [inline src/components/TracklistsCard/TracklistsCard.js]: Use !!text as strings are valid React children as well and might get rendered wrong. Same goes for title above

File: src/components/Section/Section.js (hunk 1 above)
Comment (ar7urs) [reply src/components/Section/Section.js]: fixed

File: src/components/TracklistsCard/TracklistsCard.js (hunk 2 above)
Comment (ar7urs) [reply src/components/TracklistsCard/TracklistsCard.js]: fixed

**Pull request merged**